src/maintainer/knowledge_base.rst

```diff
@@ -1403,3 +1403,11 @@ The tl;dr here is that conda sorts as follows:
 So make sure that you **tag** your package in such a way that the package name
 that conda-build spits out will sort the package uploaded with an ``rc`` label
 higher than the package uploaded with the ``dev`` label.
+
+Linking jpeg
+============
+
+If you mantain a feedstock that depends on jpeg, please make it depend on the ``jpeg`` package, not 
+on the ``libjpeg-turbo`` package. ``jpeg`` and ``libjpeg-turbo`` have common symbols and loading both into the
+same process can be problematic. It is possible that in the future conda-forge will switch to use 
+libjpeg-turbo, but at the moment depending on libjpeg-turbo is not supported and discouraged.
```

Comment:
@conda-forge/core Do we want to make a declaration of no support and discouragement? We could also add a run constraint to force envs to have one or the other but not both.

Reply:
Not really. Turbo is incredibly useful (particularly for the imaging community that heavily use it). Would much prefer us to figure out a way for turbo to be supported in conda-forge

Reply:
Until we do, having both `libjpeg` and `libjpeg-turbo` should be discouraged.

Reply:
@beckermr the kind of plubming you suggest is what we are trying to outline (And eventually do) in https://github.com/conda-forge/conda-forge.github.io/issues/673

Reply:
If we do go this route of adding this to the docs, can we reference the issue that Mark linked above and note that we would like this to change and need help from the community to move this forward?

Reply:
> 
> 
> If we do go this route of adding this to the docs, can we reference the issue that Mark linked above and note that we would like this to change and need help from the community to move this forward?

If there is consensus on this I will be happy to modify the PR.

Reply:
I think @isuruf outlined a plan at one point htat would be similar to openblas and MKL

https://github.com/conda-forge/qt-feedstock/issues/147#issuecomment-803620575

Reply:
If they are ABI compatible then we can swap them like we do for BLAS. IDK if this is true or not.

Reply:
IDK what he meant by "same symbols".

Reply:
AIUI they are iff we use JPEG 8. So there would be a need to either maintain two JPEG versions or switch to only using 8

Reply:
To sure if ther was any update on this, but I would be happy to change the PR to reflect the current situation.